File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1064.cs
using System;

class URI
{
    static void Main() {
        double valor, n=0;
        int positivos=0, i;
    
        for(i=1;i<=6;i++){
            valor = double.Parse(System.Console.ReadLine().Trim());
            if(valor>0){
                positivos++;
                n+=valor;
            }
        }
    
        Console.Write("{0} valores positivos\n{1:f1}\n", positivos, n/positivos);
        
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1064.rb
n=0
positivos=0

for i in 1..6 do
    valor = gets.to_f
    if(valor>0) then
        positivos+=1
        n=n+valor
    end
end

puts "#{positivos} valores positivos\n%0.01f\n" % (n/positivos).round(2)
